.github/workflows/production.yml

```diff
@@ -0,0 +1,30 @@
+name: Build & Production
```

Comment:
We already have a job includes the word build, but it just tests that the static build is working in general. Probably one or both should be changed to be more specific in order to avoid confusions. E.g. Build static / Build image.

Reply:
Is it ok if you change the existing static build test workflow name? As we have implemented these workflows for all projects using this naming so it would be great if all projects had unified naming.